app/src/main/java/org/oppia/android/app/help/HelpActivityPresenter.kt

```diff
@@ -32,18 +36,24 @@ class HelpActivityPresenter @Inject constructor(
   private lateinit var selectedHelpOptionTitle: String
   private var selectedDependencyIndex: Int? = null
   private var selectedLicenseIndex: Int? = null
+  private var internalPolicyPage: PolicyPage = PolicyPage.POLICY_PAGE_UNSPECIFIED
```

Comment:
Should we make `policiesActivityArguments` as global so that it can be used in `handleOnSavedInstanceState` too directly. WDYT?

Reply:
I didn't get why we need to make it global.

Reply:
Yeah we cannot make this change because of line 318